posthog/api/paths.py

```diff
@@ -26,31 +27,77 @@ def _determine_path_type(self, request):
         event: Optional[str] = "$pageview"
         event_filter = {"event":event}
         path_type = "properties->> \'$current_url\'"
+        start_comparator = "{} ~".format(path_type)
 
         # determine requested type
         if requested_type:
             if requested_type == "$screen":
                 event = "$screen"
                 event_filter = {"event":event}
                 path_type = "properties->> \'$screen_name\'"
+                start_comparator = "{} ~".format(path_type)
             elif requested_type == "$autocapture":
                 event = "$autocapture"
                 event_filter = {"event":event}
                 path_type = "tag_name_source"
+                start_comparator = "group_id ="
             elif requested_type == "custom_event":
                 event = None
                 event_filter = {'event__regex':'^[^\$].*'}
                 path_type = "event"
-        return event, path_type, event_filter
+                start_comparator = "event ="
+        return event, path_type, event_filter, start_comparator
+
+    @action(methods=['GET'], detail=False)
+    def elements(self, request: request.Request):
```

Comment:
added an endpoint off the /paths that will retrieve the top 50 elements that a user can use as a start point. could possibly belong somewhere else but we don't have a dedicated elements endpoint and this is a specific need here